common/network/iota_packet.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_NETWORK_IOTA_PACKET_H__
+#define __COMMON_NETWORK_IOTA_PACKET_H__
+
+#include "common/network/socket.h"
+#include "common/trinary/tryte.h"
+
+#define TRANSACTION_PACKET_SIZE 1650
+
+typedef struct {
+  tryte_t data[TRANSACTION_PACKET_SIZE];
```

Comment:
it's not 1650 trytes, but 8019 trits encoded as 1604 bytes + 230 trytes encoded as 46 bytes.
correct type should be char* ?